src/index.ts

```diff
@@ -259,6 +259,9 @@ export function main(argString = "") {
       ? 1
       : 0;
 
+  retrieveItem($item`seal tooth`);
+  retrieveItem($item`The Jokester's gun`);
```

Comment:
I think these should be in dailySetup, no?

Reply:
moving to be just below the half a purse. Good catch.